README.md

```diff
@@ -1,5 +1,101 @@
-# Preferences Server
-General Preferences Service
+# Personal Data Server
 
-This project is for the code and documentation of a common general preferences
-service that can be used to store, retrieve, and modify user preferences.
+Personal Data Server can be used to save, retrieve, update and delete personal user data. This project is written in
+[`node.js`](https://nodejs.org/en/) and uses [`Postgres`](https://www.postgresql.org/) as the backend database.
+
+This project supports these authentication methods:
+
+* Google Single Sign-on
+
+## Getting Started
+
+To work on the project, you need to:
+
+* Install [NodeJS and NPM](https://nodejs.org/en/download/) for your operating system
+* Clone the project from GitHub. [Create a fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+with your GitHub account, then run the following in your command line (make sure to replace `your-username` with
+your username):
+
+```bash
+git clone https://github.com/your-username/preferencesServer
+```
+
+* Obtain Google OAuth 2.0 client credentials from the [Google API Console](https://console.developers.google.com/)
+* From the root of the cloned project, run commands below to create `dataModel/SsoProvidersData.sql`
+
+```bash
+cp dataModel/SsoProvidersData.sql.example dataModel/SsoProvidersData.sql
+```
+
+* Edit `dataModel/SsoProvidersData.sql` to replace the string value at the line 17 and 18 with Google OAuth 2.0 client id
+and client secret. This SQL file will be loaded into the database later
+
+* Run the command below to start the server
+
+```bash
+npm start
+```
+
+* Open `http://localhost:3000/` in a browser demonstrates a process of Google Single Sign-on
+
+### Skip Docker
+
+By default, running `npm start` or `npm test` will start a Postgres docker container to serve the backend database.
```

Comment:
When running npm test in my Ubuntu vm I got a permission failure when trying to execute docker:

````
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/PostgresForTests/start": dial unix /var/run/docker.sock: connect: permission denied
Error: failed to start containers: PostgresForTests
````

Does something need to be added to the instructions to explain that the tests should somehow be run with elevated permissions?